README.md

```diff
@@ -88,9 +87,9 @@ FLAGS:
 
 OPTIONS:
         --subgraph [<NAME>:]<IPFS_HASH>         Name (optional) and IPFS hash of the subgraph manifest
-        --ethereum-ipc <NETWORK_NAME>:<FILE>    Ethereum network name (e.g. 'mainnet') and Ethereum IPC pipe path, separated by a ':'
-        --ethereum-rpc <NETWORK_NAME>:<URL>     Ethereum network name (e.g. 'mainnet') and Ethereum RPC endpoint URL, separated by a ':'
-        --ethereum-ws <NETWORK_NAME>:<URL>      Ethereum network name (e.g. 'mainnet') and Ethereum WebSocket endpoint URL, separated by a ':'
+        --ethereum-ipc <NETWORK_NAME>:<FILE>    Ethereum network name (e.g., 'mainnet') and Ethereum IPC pipe path, separated by a ':'
+        --ethereum-rpc <NETWORK_NAME>:<URL>     Ethereum network name (e.g., 'mainnet') and Ethereum RPC endpoint URL, separated by a ':'
+        --ethereum-ws <NETWORK_NAME>:<URL>      Ethereum network name (e.g., 'mainnet') and Ethereum WebSocket endpoint URL, separated by a ':'
         --ipfs <HOST>:<PORT>                    HTTP address of an IPFS node
         --postgres-url <URL>                    Location of the Postgres database used for storing entities
```

Comment:
Let's replace the usage instructions with the current output of `graph-node --help`:
```
USAGE:
    graph-node [FLAGS] [OPTIONS] --ethereum-ipc <NETWORK_NAME:FILE> --ethereum-rpc <NETWORK_NAME:URL> --ethereum-ws <NETWORK_NAME:URL> --ipfs <HOST:PORT> --postgres-url <URL>

FLAGS:
        --debug      Enable debug logging
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
        --admin-port <PORT>                           Port for the JSON-RPC admin server [default: 8020]
        --elasticsearch-password <PASSWORD>
            Password to use for Elasticsearch logging [env: ELASTICSEARCH_PASSWORD]

        --elasticsearch-url <URL>
            Elasticsearch service to write subgraph logs to [env: ELASTICSEARCH_URL=]

        --elasticsearch-user <USER>                   User to use for Elasticsearch logging [env: ELASTICSEARCH_USER=]
        --ethereum-ipc <NETWORK_NAME:FILE>
            Ethereum network name (e.g. 'mainnet') and Ethereum IPC pipe, separated by a ':'

        --ethereum-polling-interval <MILLISECONDS>
            How often to poll the Ethereum node for new blocks [env: ETHEREUM_POLLING_INTERVAL=]  [default: 500]

        --ethereum-rpc <NETWORK_NAME:URL>
            Ethereum network name (e.g. 'mainnet') and Ethereum RPC URL, separated by a ':'

        --ethereum-ws <NETWORK_NAME:URL>
            Ethereum network name (e.g. 'mainnet') and Ethereum WebSocket URL, separated by a ':'

        --http-port <PORT>                            Port for the GraphQL HTTP server [default: 8000]
        --ipfs <HOST:PORT>                            HTTP address of an IPFS node
        --node-id <NODE_ID>                           a unique identifier for this node [default: default]
        --postgres-url <URL>                          Location of the Postgres database used for storing entities
        --subgraph <[NAME:]IPFS_HASH>                 name and IPFS hash of the subgraph manifest
        --ws-port <PORT>                              Port for the GraphQL WebSocket server [default: 8001]
```